src/components/NavBar/SearchBarDropdown.tsx

```diff
@@ -149,34 +145,27 @@ export const SearchBarDropdown = ({
   const { pathname } = useLocation()
   const { chainId } = useWeb3React()
   const isNFTPage = useIsNftPage()
-  const isNftGraphqlEnabled = useNftGraphqlEnabled()
   const isTokenPage = pathname.includes('/tokens')
   const [resultsState, setResultsState] = useState<ReactNode>()
 
-  const { data: trendingCollectionResults, isLoading: trendingCollectionsAreLoading } = useQuery(
-    ['trendingCollections', 'eth', 'twenty_four_hours'],
-    () => fetchTrendingCollections({ volumeType: 'eth', timePeriod: 'ONE_DAY' as TimePeriod, size: 3 })
-  )
-
-  const { data: gqlData, loading } = useTrendingCollections(3, HistoryDuration.Day)
+  const { data, loading } = useTrendingCollections(3, HistoryDuration.Day)
 
   const trendingCollections = useMemo(() => {
-    const gatedTrendingCollections = isNftGraphqlEnabled ? gqlData : trendingCollectionResults
-    return gatedTrendingCollections && (!isNftGraphqlEnabled || !loading)
-      ? gatedTrendingCollections
-          .map((collection) => ({
+    return !loading
+      ? data
+          ?.map((collection) => ({
```

Comment:
Is it intentional to return `undefined` if the `data` is empty? If `!loading`, can we assert that `data` is non-empty?

Reply:
Yes, then no suggestions will load which is the desire behaviour. We can't assert `data` is non-empty if `!loading` because it is possible the query returns empty

Reply:
This must be an empty array, not null, or else L319 may err at runtime.